source/scss/_library/_objects.buttons.scss

```diff
@@ -2,29 +2,52 @@
     $BUTTONS
 \*------------------------------------*/
 
-.o-button,
 button,
 input[type="submit"] {
+  appearance: none;
+  border: none;
+  cursor: pointer;
+  background-color: transparent;
+  padding: 0;
+}
+
+.o-button {
   display: inline-flex;
   justify-content: center;
   align-items: center;
-  padding: 15px 25px;
-  line-height: 1.2;
+  padding: $space $space 18px;
+  line-height: 22px;
   color: $c-white;
   cursor: pointer;
   transition: all 0.5s ease;
   overflow: hidden;
-  font-family: $ff-font--primary;
+  font-family: $ff-font--monospace;
   text-transform: uppercase;
   border: none;
+  font-size: 18px;
   font-weight: bold;
   letter-spacing: 2px;
   text-align: center;
-  background-color: $c-tertiary;
+  background-color: $c-primary;
   appearance: none;
```

Comment:
this seems like a duplicate, since the base `button` element also has `appearance: none;` set.

Reply:
Good catch!